.drone.yml

```diff
@@ -212,8 +228,44 @@ trigger:
     - "staging-next"
   event:
     - pull_request
+
+---
+
+# On push, cache a clone of staging to drone's S3 bucket.
+# The staging cache reduces git bandwidth in PR pipelines.
+kind: pipeline
+type: docker
+name: cache-staging-clone
+
+clone:
+  disable: true
+
+steps:
+
+  - name: clone
+    image: bitnami/git
+    commands:
+      - git lfs install
+      - git clone --branch $DRONE_BRANCH --depth 100 $DRONE_GIT_HTTP_URL . 
```

Comment:
Opted to cache a full clone instead of using `--depth` fetches because the perf speedup was tiny and it increased complexity, see: https://github.com/code-dot-org/code-dot-org/pull/56389#issuecomment-1931622715